pkg/services/accesscontrol/migrator/zanzana.go

```diff
@@ -379,3 +391,49 @@ func basicRoleAssignemtCollector(store db.DB) TupleCollector {
 		return nil
 	}
 }
+
+func roleAssignemtCollector(store db.DB) TupleCollector {
```

Comment:
this one is just migrating roles that are directly assigned to a user. I think we could also add a collector for team

Reply:
Thanks, I forgot it, will add it for sure!